git.go

```diff
@@ -7,45 +7,45 @@ import (
 )
 
 // RunGitCmd shells out to git in the context of the dstask repo.
-func RunGitCmd(args ...string) error {
-	args = append([]string{"-C", GIT_REPO}, args...)
+func RunGitCmd(repoPath string, args ...string) error {
+	args = append([]string{"-C", repoPath}, args...)
 	return RunCmd("git", args...)
 }
 
 // MustRunGitCmd delegates to RunGitCmd and exits the program on any error.
-func MustRunGitCmd(args ...string) {
-	err := RunGitCmd(args...)
+func MustRunGitCmd(repoPath string, args ...string) {
```

Comment:
Quite a lot of variable passing again (I think I prefer the global variables / env, though not if it causes a testing issue)